### PR TITLE
Declare anonymous struct array

### PR DESCRIPTION
**Pull request by kant2002**

### CODE REVIEW

File: Cesium.CodeGen.Tests/verified/CodeGenArrayTests.StaticStructArrayTest.verified.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+﻿emptyString
```
Comment (ForNeVeR) [inline Cesium.CodeGen.Tests/verified/CodeGenArrayTests.StaticStructArrayTest.verified.txt:1]: Huh? What happened here? It yield no codegen results?

File: Cesium.CodeGen.Tests/verified/CodeGenArrayTests.StructArrayTest.verified.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+﻿System.Void <Module>::test()
+  IL_0000: ret
```
Comment (ForNeVeR) [inline Cesium.CodeGen.Tests/verified/CodeGenArrayTests.StructArrayTest.verified.txt:2]: This one is also not very useful. I assume we'd need to upgrade these tests to handle the global variables correctly.

File: Cesium.CodeGen.Tests/verified/CodeGenArrayTests.StaticStructArrayTest.verified.txt (hunk 1 above)
Comment (kant2002) [reply Cesium.CodeGen.Tests/verified/CodeGenArrayTests.StaticStructArrayTest.verified.txt:1]: Yes. I need to check only emitted classes. Maybe we need new type of tests.

Comment (ForNeVeR) [reply Cesium.CodeGen.Tests/verified/CodeGenArrayTests.StaticStructArrayTest.verified.txt:1]: Don't worry, I'll handle this myself.

**Pull request merged**